KWoC Templates (#5)
kossiitkgp/mailing-scripts: templates/kwoc/end-eval-reminder.txt

> KWoC : End-Evaluation Reminder!!

Reviewer: ```suggestion
<b>KWoC End-Evaluation Reminder</b>
```

> KWoC 2023 : Final Evaluation Reminder!!

Reviewer 2: ```suggestion
KWoC 2023 : End-Evaluation Reminder!!
```

Dear Participant,<br><br>

> We hope this email finds you well. Your hard work and dedication throughout the program have been truly commendable, and we greatly appreciate the efforts you have put into the project<br><br>

Reviewer: ```suggestion
We hope this email finds you well. Your hard work and dedication throughout the program have been truly commendable, and we greatly appreciate the efforts you have put into the project.<br><br>
```

> This is to remind you that the final evaluation deadline for the project in the <b>Kharagpur Winter of Code (KWoC)<b> program is fast approaching.<br><br>

Reviewer 2: ```suggestion
This is to remind you that the end-evaluation deadline for <b>Kharagpur Winter of Code (KWoC)<b> program is fast approaching.<br><br>
```

> This is to remind you that the end-evaluation deadline for <b>Kharagpur Winter of Code<b> program is fast approaching. The deadline for the end-evaluation is {end_eval_date}.<br><br>

Reviewer: I think we should mention that the participant should check their dashboard to complete the end evaluation.

> Please submit your report by {end_eval_date}, highlighting your progress, achievements, challenges, and suggestions.<br><br>

Reviewer 2: There is no such thing like report

> 

Reviewer: Same here. Is there any reason for adding these empty lines?
